src/main/js/controls/Timeline/helpers/creationHelpers.js

```diff
@@ -91,7 +91,7 @@ const getXAxisXPosition = (config) => config.padding.left;
  * @returns {number} Position for the axis
  */
 const getXAxisYPosition = (config) =>
-    (config.padding.top + config.padding.bottom) * 2;
+    config.padding.top * 1.5 + config.padding.bottom;
```

Comment:
Why are we doing this? Didnt we fix a bug for firefox recently that increased the padding top? 
My concern is that, the unit tests no longer give us the protection in this case. 
Unit tests have a number 300. Fix 1 increases top to 20 -> Unit test now checks for 320.
Fix 2 decreases top by half -> Unit test now checks for 160. i.e. this no longer tests the Fix 1

Reply:
> Why are we doing this? Didn't we fix a bug for firefox recently that increased the padding-top?

IN PR #107, we had decreased the padding-bottom from `5` to `2.5`. Which made it work for timeline but as a result, all the graphs created from graph-construct had their top Y and Y2 axis ticks getting clipped. So, I have reverted the padding-bottom back to `5` and to solve the timeline issue this change `config.padding.top * 1.5 + config.padding.bottom;` was made. As this would only affect the timeline. 
What do you think? @abhijit945

Reply:
I am good with that as long as we are only changing the unit tests for impacted functionality.